src/OrchardCore.Build/Dependencies.props

```diff
@@ -47,7 +47,7 @@
     <PackageManagement Include="NLog.Web.AspNetCore" Version="5.2.2" />
     <PackageManagement Include="NodaTime" Version="3.1.8" />
     <PackageManagement Include="OpenIddict.AspNetCore" Version="4.1.0" />
```

Comment:
You'll also want to bump this package to 4.2.0 😃

Reply:
Oops

BTW seems you added 26 new providers on 4.2.0,

Reply:
> BTW seems you added 26 new providers on 4.2.0

Yeah. That said, Orchard's OpenID module is not based on the new OpenIddict client (it uses MSFT's OIDC client), so it's not something the OC users will benefit from 😃 

If there was demand from the community, replacing the MSFT OIDC client by OpenIddict is definitely something we could consider (but it would be a breaking change, so not something to take lightly).

Reply:
If you don't mind I might need your help to build a new OpenId module based on OpenIddict on [OCC](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules)

Reply:
Sure, what do you have in mind?

Reply:
I might open an issue in OCC repo describing what's my intend in more details